style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+  padding: 4px 0;
+  width: 375px;
+}
+
+header {
+  width: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+/* Toolbox */
+
+.nav {
+  display: flex;
+  flex-flow: column wrap;
+  height: 50px;
+  align-content: space-between;
+}
```

Comment:
- [ ] Add some margin here like: `padding: 10px 12px` as given in your figma design. Using margin and paddings improves UX of project and also helps clients to understand your content easier 😃🙌